werekata.js

```diff
@@ -2,151 +2,111 @@ var flyfeatures; // once the view is loaded the array of coords is stored here,
 var leg = 0; // just to keep track fo where we are in the journey
 var flyview;
 
-
 var urlParams = new URLSearchParams(window.location.search);
 var urltoload = urlParams.get("load");
-var baselayer = "hybrid";
-if (urlParams.has("base")) {
-    baselayer = urlParams.get("base");
-}
 
-var sortoption = urlParams.get('sort');
+var sortoption = urlParams.get("sort");
 if (sortoption === "end") {
     urltoload = urltoload + "?sort=end";
 } else if (sortoption === "start") {
     urltoload = urltoload + "?sort=start";
 }
```

Comment:
This could be used for controlling the order of flying by append sort=start in URL when viewing werekata flight by time. or we could put this config from the JSON feed.

Reply:
@MufengNiu 

Don't think we need the configuration for this.

If I have a data feed for example available at http://example.com/data, does this parameter affect the output given my data feed doesn't recognize this parameter?

Or should we remove it because as a data feed provider, I control the order of features within the feature collection. The view doesn't need to know how I sorted my features, either by time, name or any other order. The view only stick to the order presented in the data feed.

Reply:
removed